app/models/blueprint.rb

```diff
@@ -101,38 +101,27 @@ def space_attributes
         {
           name: "Listed Room 1",
           publicity_level: :listed,
-          access_level: :unlocked,
-          access_code: nil,
+          access_level: :public,
           furniture_placements: {
             markdown_text_block: {content: "# Welcome!"}
           }
         },
         {
           name: "Listed Room 2",
```

Comment:
Should we rename all of these to "Public", instead of "Listed"?

Reply:
Oo, I remember why I didn't!!! If I had, it would have broken feature tests which have those names hardcoded... And technically they are *listed* they're just also *public*, since *listed* indicates whether or not they are enumerated in the navigation menu.